test/Dialect/Perf/perf-invalid.mlir

```diff
@@ -5,7 +5,7 @@ func.func @perf_no_outs(%n: i64) {
   %deltas = memref.alloc(%size) : memref<?xf64>
   %out = arith.constant 0 : i64
 
-  // expected-error @below {{'perf.bench' op failed to verify that result types match types of iter_args}}
+  // expected-error @below {{cannot name an operation with no results}}
```

Comment:
Why are we testing this? This is MLIR stuff, no?

Reply:
Not entirely. The error comes from the basic verifier, yes, but it means we're not generating the `perf.bench` correctly. Before, this was an error in the `perf` verifier, but that error was actually misplaced. This is the right place to put it.

I can remove the test, but I think it works well to encode expectation, even if the error is somewhere else. Doesn't cost us much either.